src/Tribe/Aggregator/API/Abstract.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+abstract class Tribe__Events__Aggregator__API__Abstract {
+	public $service;
+	public $cache_group = 'tribe_ea';
+
+	public function __construct( Tribe__Events__Aggregator__Service $service ) {
```

Comment:
Can we avoid passing the variable in the construct? Since the Service class will always be the same, it would make more sense to setup it up internally rather then passing as a param.